starters/README.md

```diff
@@ -1,47 +1,30 @@
 # Starters
 
-This folder stores "starter" projects for the CLI. The idea is that during the CLI execution, the developer can choose a particular starter app and combine it with a specific server and features.
+This folder stores "starter" projects for the cli. The idea is that during the cli execution, the developer can choose a particular starter app and combine it with a specific server and features.
 
 All starters are based on `starters/apps/base`, including the `package.json` and `tsconfig.json`. Depending on the options the user selects, their starter merges into the `base` app.
 
 ## Developer
 
-Here are steps to try out the CLI in a local environment.
+Here are steps to try out the cli in a local environment.
 
-1. Build CLI:
+1. Build the cli:
 
-   ```
-   # pnpm build.cli
-   ```
-
-1. Run CLI:
-
-   ```
-   # node ./packages/create-qwik/dist/create-qwik.cjs
-   💫 Let's create a Qwik project 💫
-
-   ✔ Project name … todo-express
-   ✔ Select a starter › Todo
-   ✔ Select a server › Express
+```zsh
+pnpm build.cli
+```
 
-   ⭐️ Success! Project saved in todo-express directory
+2. Run the cli:
 
-   📟 Next steps:
-   cd todo-express
-   npm install
-   npm start
-   ```
+```zsh
+pnpm cli.qwik
+```
 
-1. Change to generated location
-   ```
-   cd todo-express
-   npm install
-   npm start
-   ```
+> If you want to test the cli on consumer repository, you can `pnpm link.dist` on the qwik monorepo, then `pnpm link --global @builder.io/qwik` on the consumer project, and finally run the cli from there.
 
-## Publishing `create-qwik` CLI Package
+## Publishing `create-qwik` cli Package
```

Comment:
you changed cli to lowercase but if I'm not wrong, this should be uppercase because it's an acronym. :thinking:

Reply:
Hmmm, I prefer lowercase feels more modern to me. I think it's not a big deal as this is an internal readme anyways. I'll merge as I'm too lazy to bring it back 👅